### PR TITLE
Adding support for rails 5 and 6

### PR DESCRIPTION
**Pull request by sachin**

### CODE REVIEW

File: lib/serial_preference/preference_definition.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def required?
     end
 
     def numerical?
-      @column.number?
+      [:integer, :float, :decimal].include?(@column.type)
```
Comment (gaurish) [inline lib/serial_preference/preference_definition.rb]: will this work with rails 4 and rails 6 both?

File: lib/serial_preference/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module SerialPreference
-  VERSION = "1.2.1"
+  VERSION = "1.2.2"
```
Comment (gaurish) [inline lib/serial_preference/version.rb]: this should be 1.3 not a patch version.

File: lib/serial_preference/preference_definition.rb (hunk 1 above)
Comment (sachin) [reply lib/serial_preference/preference_definition.rb]: yes

File: lib/serial_preference/version.rb (hunk 2 above)
Comment (sachin) [reply lib/serial_preference/version.rb]: done

**Pull request merged**